core/templates/dev/head/components/answer_group_editor_directive.html

```diff
@@ -92,4 +92,11 @@
                   outcome="outcome"
                   suppress-warnings="suppressWarnings()">
   </outcome-editor>
+
+  <br ng-if="!defaultAnswerGroup">
+  
+  <div ng-if="!defaultAnswerGroup">
+        <correct-editor is-editable="isEditable" on-save-correct="getOnSaveAnswerGroupCorrectFn()" labelled-as-correct="labelledAsCorrect">
```

Comment:
Following the rest of the file, indent this tag by 2 spaces from the previous tag.

Reply:
Done!